src/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/gather_weights_decompression.hpp

```diff
@@ -70,13 +50,6 @@ class GatherWeightsDecompression : public testing::WithParamInterface<GatherWeig
                                              const bool add_subtract,
                                              const bool reshape_on_decompression,
                                              const bool per_tensor_zp);
-    std::shared_ptr<ov::Node> init_compressed_weights_subgraph(const ov::Shape& data_shape,
-                                                               const int group_size,
-                                                               const ov::element::Type data_precision,
-                                                               const ov::element::Type output_precision,
-                                                               const bool add_subtract,
-                                                               const bool reshape_on_decompression_constant,
-                                                               const bool per_tensor_zp);
```

Comment:
This builder is moved to weights_decompression_builders.hpp and renamed to `initGatherDecompressionSubgraph`